Client/Frontend/Fakespot/Views/FakespotLoadingView.swift

```diff
@@ -0,0 +1,85 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import UIKit
+
+class FakespotLoadingView: UIView, ThemeApplicable {
+    private enum UX {
+        static let card1And3Height: CGFloat = 192
+        static let card2Height: CGFloat = 40
+        static let card4And5Height: CGFloat = 80
```

Comment:
Could this be small, medium and large card height? Instead of the card numbers. I could see this misused in the long run (i.e. card2Height used on card 3 height) if we're asked to change the card height order